scripts/publish_website.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# Copyright 2019 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script builds the current version of the H3 documentation website
+# and pushes it to production. This assumes that you have set the remote
+# for gh-pages to the upstream (i.e. uber/h3) remote.
+
+git checkout gh-pages
+git pull
```

Comment:
Nit: `git pull origin gh-pages` for those that have the newer default config to not guess which remote and branch to pull from.

Reply:
Well, part of the issue here is that my remote in this context is actually `upstream`, not `origin` - `origin` is my fork on GH, `upstream` is the `uber/h3` repo. Just using `git pull` and requiring the caller to set the upstream seemed like a reasonable way not to hard-code this (and git will prompt you to set the upstream if not set).